src/main/webapp/main_hub.html

```diff
@@ -13,29 +13,18 @@
       <a href="project.html" id="project">Project Page</a>
       <a href="task.html" id="task">Task Page</a>
     </div>
-
-    <!-- 'projects' can be later styled to be a flex box for grid display-->
-    <div id="projects">
-      <a href="project.html">
-        <div class="projects-board">
-          <h3 class="header">Project A</h3>
-        </div>
-      </a>
-      <a href="project.html">
-        <div class="projects-board">
-          <h3 class="header">Project B</h3>
-        </div>
-      </a>
-      <a href="project.html">
-        <div class="projects-board">
-          <h3 class="header">Project C</h3>
-        </div>
-      </a>
-      <a href="project.html">
-        <div class="projects-board">
-          <h3 class="header">Create or Join Project</h3>
-        </div>
-      </a>
+    <div id="content">
+      <h1 id="home-header">My Projects</h1>
+      <div class="new-project">
+        <!-- TODO: implement goToProject() method in js for buttons to work -->
+        <button onclick="goToProject(1)">Create Project</button>
+        <button onclick="goToProject(2)">Join Project</button>
+      </div>
+      <div class="projects">
+        <button onclick="goToProject(2)">Project A</button>
+        <button onclick="goToProject(3)">Project B</button>
+        <button onclick="goToProject(4)">Project C</button>
```

Comment:
Fyi I only created two mock projects, so feel free to add more projects/tasks if you want `goToProject(3)` and `goToProject(4)` to work. Alternatively, if you leave it blank like `goToProject()`, it'll pick a random project to display.